.github/ISSUE_TEMPLATE/new-feature-template.md

```diff
@@ -7,4 +7,6 @@ assignees: ''
 
 ---
 
-<Describe the feature request>
+## Feature Request
```

Comment:
Do we really need a heading here? We've already got the feature tag auto-added and the issue starts with `Feature:`, also saying `## Feature request` at the top seems unneeded (to me).

Maybe just change from `<>` to `_` and leave the rest of this one as-is?